lms/templates/register-form.html

```diff
@@ -264,5 +269,5 @@ <h2 class="sr">${_("Account Acknowledgements")}</h2>
 </div>
 
 <div class="form-actions">
-  <button name="submit" type="submit" id="submit" class="action action-primary action-update register-button">${_('Register')} <span class="orn-plus">+</span> ${_('Create My Account')}</button>
+  <button name="submit" type="submit" id="submit" class="action action-primary action-update register-button">${Text(_('Register'))} <span class="orn-plus">+</span> ${Text(_('Create My Account'))}</button>
```

Comment:
Same comment.  You don't need to use `Text` to wrap plain-text if there is **no** HTML being formatted into it.